src/effect.cpp

```diff
@@ -594,6 +600,7 @@ void UpdateEffects()
                     if(e.Frame >= 4)
                         e.Frame = 0;
                 }
```

Comment:
Whoops, this block needs to be slightly moved left, otherwise, a mess in the code style...